client/src/store/index.js

```diff
@@ -1945,22 +1945,22 @@ export const mutations = {
     state.nrData.lastUpdate = state.lastUpdate
 
     if (state.consentDateForEdit) {
-      let date = new moment(state.consentDateForEdit + ' 00:01:00')
+      let date = new moment(state.consentDateForEdit + ' 23:59:00')
```

Comment:
In the ticket (10246) it mentions that consent date might need to be updated. Did you confirm with Linda? And I am not sure where else this date is used.

Reply:
consent date needs to be updated as well.  Linda chatted with Debbie, and they want the expiry dates to be on the previous day at 23:59.    2 minutes before the 00:01:00.   so i need to fix this in the update.

Reply:
Wait, what? I just chatted with Linda, who said the consent time doesn't matter, just the date.

As for expiry date... the previous day? What do you mean -- what needs to change?